config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get "/" => "static_pages#home"
+  get "/about" => "static_pages#about"
+  get "/contact" => "static_pages#contact"
```

Comment:
please create a proper `_header.html.erb` inside the `layouts` so it will be easier to access the other pages